gems/pending/util/miq-xml.rb

```diff
@@ -5,43 +5,34 @@
 require 'util/xml/xml_diff'
 require 'util/xml/xml_patch'
 
-class MiqXml  
-	#MIQ_XML_VERSION = 1.0
-	#MIQ_XML_VERSION = 1.1	# Added create_time to root in seconds for easier time conversions
-	MIQ_XML_VERSION = 2.0	# Changed sub-xmls, added namespaces
+class MiqXml
+  MIQ_XML_VERSION = 2.0 # Changed sub-xmls, added namespaces
 
-	@@defaultXmlType = :rexml  # REXML is always available so default it here.
+  @@defaultXmlType = :rexml  # REXML is always available so default it here.
 
-  # Now test to see if nokogiri is available
+  # Set to true if nokogiri is set as parser.
   @@nokogiri_loaded = false
-	begin

```

Comment:
@Fryguy is correct that this was just a way to determine if Nokogiri was available in the environment we were running in at the time.  The external SmartProxy for Windows and Linux were not always built with that library.

Since it is always available can we just remove the `@@nokogiri_loaded` completely?